### PR TITLE
menu page

### PR DESCRIPTION
**Pull request by MikeDThorpe**

### CODE REVIEW

File: src/pages/Menu.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from "react";
+import {Menus} from '../components';
 
 function Menu(props) {
   return (
     <div class="container">
-      <h1>Menu</h1>
+      <h1 className="py-5">Menu</h1>
```
Comment (matthewcotton) [inline src/pages/Menu.js:7]: Did you mean for this title to still be included? Asking as it's not in the wireframe.

**Review by matthewcotton — APPROVED** [review]
Very good. Everything works great. I like the addition of the booking drop down.

File: src/pages/Menu.js (hunk 1 above)
Comment (MikeDThorpe) [reply src/pages/Menu.js:7]: Thanks for the feedback - the H1 is  not in the wireframe but happy for it to stay in if you are.

Comment (matthewcotton) [reply src/pages/Menu.js:7]: yes sure

**Pull request merged**